paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/sink/CommitterOperatorTest.java

```diff
@@ -246,7 +256,7 @@ private OneInputStreamOperatorTestHarness<Committable, Committable> createLossyT
                 new CommitterOperator(
                         true,
                         initialCommitUser,
-                        user -> new StoreCommitter(table.newCommit(user)),
+                        user -> new StoreCommitter(table.newStreamWriteBuilder().newCommit()),
```

Comment:
You forgot `withCommitUser`. I'm thinking if we need to force users to specify a `commitUser` when using `StreamWriteBuilder`, instead of generating a random UUID for them. @JingsongLi

Reply:
Thank you for you reply, I will fix this problem later.